src/instruments/src/EWD/elements/N1.tsx

```diff
@@ -0,0 +1,330 @@
+                                showValue
+                                textNudgeY={6}
+                                textNudgeX={-13}
+                                multiplierInner={0.9}
+                            />
+                            <GaugeMarkerComponent
+                                value={11}
+                                x={x}
+                                y={y}
+                                min={min}
+                                max={max}
+                                radius={radius}
+                                startAngle={startAngle}
+                                endAngle={endAngle}
+                                className="GaugeText Gauge RedLine"
+                                multiplierInner={0.9}
+                            />
+                            <rect x={x - 19} y={y + 19} width={96} height={30} className="DarkGreyBox" />
+                            {/* N1 max limit */}
+                            <GaugeMarkerComponent
+                                value={Math.abs(N1ThrustLimit / 10)}
+                                x={x}
+                                y={y}
+                                min={min}
+                                max={max}
+                                radius={radius}
+                                startAngle={startAngle}
+                                endAngle={endAngle}
+                                className="GaugeThrustLimitIndicator Gauge"
+                            />
+                            <GaugeMaxComponent
+                                value={Math.abs(N1ThrustLimit / 10)}
+                                x={x}
+                                y={y}
+                                min={min}
+                                max={max}
+                                radius={radius}
+                                startAngle={startAngle}
+                                endAngle={endAngle}
+                                className="GaugeThrustLimitIndicatorFill Gauge"
+                            />
+                            <GaugeMarkerComponent
+                                value={N1Percent <= N1Idle ? N1Idle / 10 : N1Percent / 10}
+                                x={x}
+                                y={y}
+                                min={min}
+                                max={max}
+                                radius={radius}
+                                startAngle={startAngle}
+                                endAngle={endAngle}
+                                className="GaugeIndicator Gauge"
+                                indicator
+                            />
+                            <AvailRev x={x} y={y} mesg={availRevText} visible={availRevVisible} active={active} revDoorOpen={revDoorOpenPercentage} />
+                            <N1CommandAndTrend
+                                N1Actual={N1Percent <= N1Idle ? N1Idle / 10 : N1Percent / 10}
```

Comment:
This seems fishy... why the < idle check? The command and trend should simply be not visible if a/thr is not active or the command label is not valid.

Reply:
Likely just a copy and paste from the N1 indicator. I'll take a look and update.